pkg/auth/providers/azure/clients/ad_graph_client.go

```diff
@@ -101,7 +101,7 @@ func (c *azureADGraphClient) LoginUser(_ *v32.AzureADConfig, _ *v32.AzureADLogin
 	userPrincipal.Me = true
 	logrus.Debug("[AZURE_PROVIDER] Completed getting user info from AzureAD")
 
-	userGroups, err := c.ListGroupMemberships(GetPrincipalID(userPrincipal))
+	userGroups, err := c.ListGroupMemberships(GetPrincipalID(userPrincipal), "")
```

Comment:
Just a question, should the filter be applied here as well or is the "" intentional?

Reply:
The AD graph client is the old client.
It does not support filtering.
When you look into the function called here you will see it ignoring the filter argument.
So, yes, intentional.